Tuesdays/Week-Class/Inheratance/Assets/Scripts/PlayerController.cs

```diff
@@ -55,6 +60,8 @@ public class PlayerController : MonoBehaviour{
 		float x = CurrCharacter.Speed * _directionX;
 		float y = CurrCharacter.Speed * _directionY;
 		_rb2D.velocity = new Vector2(x, y);
+		PlayerMovementHistory.Push(_rb2D.position);
```

Comment:
for more "fun factor" I would of saved only if the distance from the last step is big enough